packages/paste-website/src/pages/components/visual-picker/index.mdx

```diff
@@ -0,0 +1,418 @@
+
+## Usage guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/visual-picker - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {VisualPickerRadioGroup, VisualPickerRadio} from '@twilio-paste/core/visual-picker';
+
+const VisualPickerExample = () => {
+  return <></>;
+};
+```
+
+#### VisualPickerRadioGroup props
+
+| Prop               | Type                                | Description                                                                               | Default                     |
+| ------------------ | ----------------------------------- | ----------------------------------------------------------------------------------------- | --------------------------- |
+| children           | `React.ReactNode`                   | VisualPickerRadio                                                                         |                             |
+| orientation?       | `oneOf(['vertical', 'horizontal'])` |                                                                                           | 'vertical'                  |
+| name               | `string`                            | Provides name for the VisualPickerRadioGroup and VisualPickerRadio(s)                     |                             |
+| value              | `string`                            | Set value equal to the value of the currently selected VisualPickerRadio                  |                             |
+| onChange           | `(value: string) => void`           |                                                                                           |
+| i18nRequiredLabel? | `string`                            | Label text for the required dot in the legend                                             | '(required)'                |
+| element?           | `string`                            | Overrides the default element name to apply unique styles with the Customization Provider | 'VISUAL_PICKER_RADIO_GROUP' |
+
+#### VisualPickerRadio props
+
+| Prop      | Type              | Description                                                                               | Default               |
+| --------- | ----------------- | ----------------------------------------------------------------------------------------- | --------------------- |
+| children  | `React.ReactNode` |                                                                                           |                       |
+| value     | `string`          | Pass to VisualPickerRadioGroup `value` when checked                                       |                       |
+| labelText | `string`          | Label text for the radio                                                                  |                       |
+| element?  | `string`          | Overrides the default element name to apply unique styles with the Customization Provider | 'VISUAL_PICKER_RADIO' |
```

Comment:
thought: missing onChange/onClick events I believe.  I guess we can just specify above the table that it extends from html input attributes